### DISCUSSION TITLE
Generic GitOps Icon?

### DISCUSSION THREAD

**Discussion by gnunn1** [open-gitops/project#159] [General]
Has there been any discussion in the past about having an approved generic GitOps icon that can be used to represent the methodology or as an alternative for product specific logos?

I found this one and am using it in a couple of slides for my presentation at GitOpsCon on Monday. It looks like it is licensed under Creative Commons and thus is not encumbered as far as I can tell but I'm no license expert.

https://www.giantswarm.io/blog/free-gitops-icon

**Comment by christianh814** [comment]
I think it's mainly "use what you'd like" but it would be good to have some sort of guidance. The thing with the Giant Swarm logo is that it incorporates the "git" logo, and the principles actually don't use the word "git" on purpose. What happens when everyone decides to use OCI? S3 Buckets? Anything else that's compliant?

I, personally, like the logos that do something along the lines of 🔄, which accurately describe the nature of the reconcile loop.

**Reply by niklasmtj** [reply]
Maybe this is some kind of a chicken and egg problem. With the word `gitops` this somehow implies using `git`. I already thought about that for some time if this make it harder for people to see a different place of storing configuration in a different medium. 

But I'm with you that the logo should reflect the loop (and so the reconciliation). +1 on that

**Reply by harrietgrace** [reply]
Even if folks do end up moving away from Git, it could be nice to retain that in the name/logo as a historical point of interest?

**Comment by gnunn1** [comment]
While I agree it from a principles point of view it doesn't need to be tied to git the methodology is called "GitOps" so from that perspective I feel like the Giant Swarm icon is fine. Just my two cents.

**Comment by cwstrommer** [comment]
I was initially skeptical of the Giant Swarm icon, but now that I think about it it's got some merit to it:

* Cog - automation
* Incorporation of the git logo - I agree with Christian that we don't necessarily want to associate this with Git specifically, but it is accepted as a de-facto standard for declarative configuration.  I would however welcome suggestions for alternatives, as I feel that it may put too much emphasis on git as a tool vs what it represents (versioned change management)

Christian's suggestion of wanting a cyclic arrow to represent a reconciliation loop is good too, but I think if we make it a spiral instead of a cycle we can also denote that we're pulling desired state, in which case we could maybe drop the git part of the logo. 

TL;DR: How about a cog+spiral theme?

**Comment by joshgav** [comment]
+1 to the suggestion that a graphic about GitOps should convey the notion of cyclic observation and reconciliation to desired state.

"git" is just for buzz as I hope we all know, it's any version-controlled file system.

**Reply by niklasmtj** [reply]
Your second point is actually a big question in my mind. Folks joining the space and learning about it might be "surprised" that there is more than just "git" to implement it. But on the other hand this might also be okay that we use "git" as the entrypoint and doing the more "advanced" stuff (whereever it takes us :D) when folks have more experience in that space.

**Reply by joshgav** [reply]
+1 @niklasmtj, it's easier to say "git" and then explain later if needed.

We could include some symbol of a [DAG](https://en.m.wikipedia.org/wiki/Directed_acyclic_graph#Genealogy_and_version_history) to represent a git commit tree, reflecting the importance of version control.
